packages/itmat-interface/src/server/router.ts

```diff
@@ -198,8 +198,9 @@ export class Router {
                     preq.end();
                 }
             },
-            onProxyReqWs: function (preq) {
-                preq.setHeader('authorization', `Basic ${Buffer.from('admin:token').toString('base64')}`);
+            onProxyReqWs: function (preq, req) {
+                const data = (req.user as IUser).username + ':token';
```

Comment:
I don't think this will work because cookie session middleware does not run on upgraded HTTP connections. Correct me if I'm wrong, but this here will just lead you to `undefined:token`.